testutil/testnode/full_node_test.go

```diff
@@ -75,10 +79,31 @@ func (s *IntegrationTestSuite) Test_Liveness() {
 
 func (s *IntegrationTestSuite) Test_PostData() {
 	require := s.Require()
-	_, err := s.cctx.PostData(s.accounts[0], namespace.RandomMessageNamespace(), tmrand.Bytes(100000))
+	_, err := s.cctx.PostData(s.accounts[0], flags.BroadcastBlock, namespace.RandomMessageNamespace(), tmrand.Bytes(100000))
 	require.NoError(err)
 }
 
 func TestIntegrationTestSuite(t *testing.T) {
 	suite.Run(t, new(IntegrationTestSuite))
 }
+
+func (s *IntegrationTestSuite) Test_FillBlock() {
+	require := s.Require()
+
+	for squareSize := 16; squareSize < appconsts.MaxSquareSize; squareSize *= 2 {
+		resp, err := s.cctx.FillBlock(squareSize, s.accounts, flags.BroadcastAsync)
+		require.NoError(err)
+
+		err = s.cctx.WaitForNextBlock()
+		require.NoError(err)
+
+		res, err := testutil.QueryWithOutProof(s.cctx.Context, resp.TxHash)
```

Comment:
[nit][optional] based on @SweeXordious 's comment

```suggestion
		res, err := testutil.QueryWithoutProof(s.cctx.Context, resp.TxHash)
```

this implies a rename in a function not defined in this PR so it can be done in a separate PR

Reply:
https://github.com/celestiaorg/celestia-app/issues/869